### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by ddagruma**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -52,28 +51,27 @@ Also note that you are free to include any other publicly available technologies
...
 
-The transformed output should be streamed to a table `SportClassYearProfileViewSummary` on the the PostgreSQL database `datalake`.  
+The transformed output should be streamed to a table `SportClassYearProfileViewSummary` on the PostgreSQL database `datalake`.  
 
 
 
 ## Deliverables
 
- *  your application should be able to be run as a stand alone console application, docker container, or from inside Visual Studio.
+ *  your application should be able to be run as in the Visual Studio debugger, or as a stand alone console application, or as a docker container.
```
Comment (steffyd) [inline readme.md:71]: run as in the Visual Sutdio Debugger should just be run in the Visual Studio debugger

File: readme.md [hunk 2]
```diff
@@ -173,7 +171,37 @@ This container simulates the OLTP and event log activity in the docker-compose e
 *  creates new event log messages in `fl-eventlogs` via procedure `CreateAthleteProfileViewEvent`
 
 
-The service can be controlled by a table `testControl` in `fl-central`.
+The service can be controlled by a table `testControl` in `fl-central`.  When the service is started, the default settings are:
+
+** Athlete **
+*  StartAthleteCount: 1000
```
Comment (steffyd) [inline readme.md:177]: Is there any limits on either of these tables?

File: readme.md (hunk 1 above)
Comment (ddagruma) [reply readme.md:71]: kk

File: readme.md (hunk 2 above)
Comment (ddagruma) [reply readme.md:178]: there are.  i will enumerate them as well.  its in the DDL

**Pull request merged**